File: Demo.Web.Models/AccountRegisterModel.cs
namespace Demo.Web.Models
{
    public class AccountRegisterModel
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int Age { get; set; }
        public string LoginEmail { get; set; }
        public string Street { get; set; }
        public string Hausnumber { get; set; }
        public string ZipCode { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string CreditCard { get; set; }
        public string ExpirationDate { get; set; }
        public bool IsActive { get; set; }
        public string Password { get; set; }
    }
}

File: Demo.Web.Client/Binding/Models/AccountRegisterModel.js

(function (os) {
    var AccountRegisterModelStep1 = function () {

        var self = this;

        self.FirstName = '';
        self.LastName = '';
        self.Age = '';
        self.Street = '';
        self.City = '';
        self.Hausnumber = '';
        self.State = '';
        self.Street = '';
        self.ZipCode = '';

        self.Initialized = false;
    }
    os.AccountRegisterModelStep1 = AccountRegisterModelStep1;
}(window.Demo));

(function (os) {
    var AccountRegisterModelStep2 = function () {

        var self = this;

        self.LoginEmail = '';
        self.Password = '';
        self.PasswordConfirm = '';
        self.IsActive = true;

        self.Initialized = false;
    }
    os.AccountRegisterModelStep2 = AccountRegisterModelStep2;
}(window.Demo));

(function (os) {
    var AccountRegisterModelStep3 = function () {

        var self = this;

        self.CreditCard = '';
        self.ExpirationDate = '';
        
        self.Initialized = false;
    }
    os.AccountRegisterModelStep3 = AccountRegisterModelStep3;
}(window.Demo));
